packages/gatsby-mdx/utils/babel-parse-to-ast.js

```diff
@@ -0,0 +1,63 @@
+// pulled from https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/utils/babel-parse-to-ast.js
```

Comment:
Pulled directly from Gatsby. Maybe we can import this directly once Gatsby 2.0 gets out of beta, since I don't think this exists in all version of Beta Gatsby 2.0